vue/components/ui/organisms/details-expandable/details-expandable.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+    <div class="details-expandable">
+        <div v-bind:class="isExpanded[sectionName] ? 'details-expandable-section-show' : 'details-expandable-section-hide'" v-for="(section, sectionName, index) in data" v-bind:key="index">
+            <div class="details-expandable-section" v-on:click="toggleSection(sectionName)">
+                <h2 class="details-expandable-header">{{ sectionName }}</h2>
+                <icon class="details-expandable-caret" v-bind:icon="isExpanded[sectionName] ? 'chevron-up' : 'chevron-down'" />
+            </div>
+            <div class="details-expandable-row" v-for="(value, name, subIndex) in section" v-bind:key="subIndex">
+                <label-ripe class="details-expandable-title" v-bind:text="capitalizeName(name)" v-bind:font-size="labelFontSize" />
```

Comment:
Maybe we should not capitalize the name in the component and instead let the component user capitalize the name if they want to (or apply any other transformation)

Reply:
here I would at least use the name capitalization as the default as this names serves as a kind of title:
![image](https://github.com/ripe-tech/ripe-components-vue/assets/30175756/7590a348-3588-4039-a91d-e665ea1eec53)

Reply:
I created a new slot where the user is free to capitalize it how it wants. If he doesn't use it, then it will capitalize the first letter by default

Reply:
Ok, seems a good approach